packages/code-snippet/src/ExpandableComponent.tsx

```diff
@@ -30,6 +30,7 @@ const UP_ICON_CLASS = 'elyra-upArrow-icon';
 const ACTION_BUTTON_CLASS = 'elyra-expandableContainer-button';
 const TITLE_CLASS = 'elyra-expandableContainer-title';
 const ACTION_BUTTONS_WRAPPER_CLASS = 'elyra-expandableContainer-action-buttons';
+const ACTION_BUTTON_COMMON_CLASS = 'elyra-expandableContainer-actionButton';
```

Comment:
I think this should be called `ACTION_BUTTON_CLASS` and that `ACTION_BUTTON_CLASS` should change to  `BUTTON_CLASS` and `BUTTON_CLASS` to `ELYRA_BUTTON_CLASS`. This would make the names consistent with their usage as well as generally more readable.